sct_ssh.py

```diff
@@ -115,7 +115,7 @@ def other(event):
     return question.ask()
 
 
-def select_instance_group(region: str = None, **tags) -> dict | None:
+def select_instance_group(region: str = None, **tags) -> list:
```

Comment:
is this related somehow to the show-monitor fix?

Reply:
yes,  without this fix it won't work.

i.e. there a bug here that if this list return only once instance, it didn't returned a list, and the command would iterate on it's keys, and not on the node details, and would fail...